lib/faker/default/food.rb

```diff
@@ -108,6 +108,19 @@ def measurement
       def metric_measurement
         fetch('food.metric_measurements')
       end
+
+      ##
+      # Retrieves ethnic category
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Food.ethnic_category #=> "Indian cuisine"
+      #
+      # @faker.version 2.18.0
```

Comment:
Can you update to the below?

```suggestion
      # @faker.version next
```

Reply:
Done